app/src/main/java/com/fstyle/androidtrainning/screen/searchoffline/SearchOfflinePresenter.java

```diff
@@ -1,5 +1,14 @@
 package com.fstyle.androidtrainning.screen.searchoffline;
 
+import android.content.Context;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports